apps/src/aiDifferentiation/AiDiffBotMessageFooter.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import Button from '@cdo/apps/componentLibrary/button/Button';
+import {commonI18n} from '@cdo/apps/types/locale';
+
+import {ChatTextMessage} from './types';
+
+import style from './ai-differentiation.module.scss';
+
+interface Props {
+  message: ChatTextMessage;
+}
+
+const AiDiffBotMessageFooter: React.FC<Props> = ({message}) => {
+  return (
+    <div className={style.messageFeedbackContainer}>
+      <div className={style.messageFeedbackLeft}>
+        <Button
+          onClick={() => {}}
+          color="white"
+          size="xs"
+          isIconOnly
+          icon={{iconStyle: 'regular', iconName: 'copy'}}
+          type="primary"
+          className={style.messageFeedbackButton}
+        />
+        <Button
+          onClick={() => {}}
+          color="white"
+          size="xs"
+          isIconOnly
+          icon={{iconStyle: 'regular', iconName: 'file-export'}}
+          type="primary"
+          className={style.messageFeedbackButton}
+        />
+      </div>
+      <div className={style.messageFeedbackRight}>
+        {commonI18n.aiFeedbackQuestion()}
+        <Button
+          onClick={() => {}}
+          color="white"
+          size="xs"
+          isIconOnly
+          icon={{iconStyle: 'regular', iconName: 'thumbs-up'}}
+          type="primary"
+          className={style.messageFeedbackButton}
+        />
+        <Button
+          onClick={() => {}}
+          color="white"
+          size="xs"
+          isIconOnly
+          icon={{iconStyle: 'regular', iconName: 'thumbs-down'}}
+          type="primary"
+          className={style.messageFeedbackButton}
+        />
+        <Button
+          onClick={() => {}}
+          color="white"
+          size="xs"
+          isIconOnly
+          icon={{iconStyle: 'regular', iconName: 'flag-pennant'}}
+          type="primary"
+          className={style.messageFeedbackButton}
+        />
+      </div>
```

Comment:
AI Tutor uses this [component](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/aiTutor/views/AssistantMessageFeedback.tsx) to handle 👍 👎 feedback, which is somewhat duplicative. I think ideally we'd have one set of feedback buttons that would take callbacks and conditionally hide/show the copy, download and flag buttons based on the context they're being rendered in. 

That being said, AI Tutor work is currently on [pause](https://codedotorg.slack.com/archives/C06KYAJDB24/p1726681936776969), so I don't know how much we want to invest in consolidating or making the UI match. 🤷‍♀️